### PR TITLE
Add derby tests to the set of External Tests running at Adopt

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Add derby tests to the set of External Tests running at Adopt
Signed-off-by: Mesbah Alam <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Related issue: #172

**Comment by Mesbah-Alam** [comment]
We can not use derby from its master branch (trunk) as it requires Java 9 or above to build : 
/https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/test.personal/105/console 

```17:00:33 BUILD FAILED
17:00:33 /derby/build.xml:182: Compiler level must be Java 9 or later.```

We need to checkout from a branch that supports Java 8 and above, which is `10.14`-- as stated in [the downloads page](http://db.apache.org/derby/derby_downloads.html)

File: thirdparty_containers/derby/playlist.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>derby_test_junit_all</testCaseName>
+	<command>docker run --rm -v $(JDK_HOME):/java  adoptopenjdk-derby-test:latest ; \
+		$(TEST_STATUS)</command>
+		<subsets>
+			<subset>SE80</subset>
```
Comment (smlambert) [inline thirdparty_containers/derby/playlist.xml]: If you drop this subset, then do you not get past the "needs Java9+" error msg?

Comment (Mesbah-Alam) [reply thirdparty_containers/derby/playlist.xml]: Hi @ShelleyLambert , yes we would, but I was under the impression that we want to test Java 8 and above with the external tests due to the fact that we have only two Java 8 builds for external tests at Adopt (openjdk8_hs* and openjdk8_j9*). Please correct me if I am wrong.

Comment (smlambert) [reply thirdparty_containers/derby/playlist.xml]: As per our slack conversation just now, we can add openjdk9* and openjdk10* builds at any time, I just had not gotten to it yet :)  

So for now, you could pull material from HEAD for Derby, and run them against Java9 SDKs

Comment (Mesbah-Alam) [reply thirdparty_containers/derby/playlist.xml]: Yes, I am on it. Thanks for clarifying.

Comment (Mesbah-Alam) [reply thirdparty_containers/derby/playlist.xml]: Updated Derby test scripts to pull Derby from HEAD. Tested here: https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/test.personal/135/console

**Pull request merged**